data_safe_haven/functions/strings.py

```diff
@@ -114,6 +115,11 @@ def sha256hash(input_string: str) -> str:
     return hashlib.sha256(input_string.encode("utf-8")).hexdigest()
 
 
+def strip_ansi(text: str) -> str:
+    """Strip all ANSI formatting from a string"""
+    return str(Text.from_ansi(text))
```

Comment:
From [this StackOverflow post](https://stackoverflow.com/questions/14693701/how-can-i-remove-the-ansi-escape-sequences-from-a-string-in-python) we could also use:

```suggestion
    ansi_escape = re.compile(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])')
    return ansi_escape.sub("", text)
```